docs/framework/install/troubleshoot-blocked-installations-and-uninstallations.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Troubleshoot blocked .NET Framework installations and uninstallations"
+description: "Troubleshoot issues you encounter that prevent or block installation of the .NET Framework. Consult blocking messages for information to resolve issues."
```

Comment:
isn't prevent and block communicating the same thing? I know the article uses it, but the term "blocking message" seems odd and I don't think it's common. Perhaps status is better.

```suggestion
description: "Troubleshoot issues you encounter that prevent installation of the .NET Framework. Consult status messages for information to resolve issues."
```